src/operator/optimizer_op.cc

```diff
@@ -105,9 +106,35 @@ It updates the weights using::
 .add_argument("var", "NDArray-or-Symbol", "Moving variance")
 .add_arguments(AdamParam::__FIELDS__());
 
+
 NNVM_REGISTER_OP(rmsprop_update)
-.describe(R"code(Update function for RMSProp optimizer. The RMSProp code follows the version in
-http://www.cs.toronto.edu/~tijmen/csc321/slides/lecture_slides_lec6.pdf Tieleman & Hinton, 2012.
+.describe(R"code(Update function for `RMSProp` optimizer.
+
+The `RMSprop` is a form of stochastic gradient descent where the gradients are
+divided by a running average of their recent magnitude. `RMSprop` was developped
```

Comment:
@zackchase isnt the cache growing with square root of the sum of squares of recent gradients?